tangerine/installation.cpp

```diff
@@ -17,20 +17,39 @@
 
 #if !_WIN64
 #include <unistd.h>
+#include <sys/types.h>
+#include <pwd.h>
+#include <iostream>
+#include <cerrno>
+#include <cstring>
 #endif
 
+namespace fs = std::filesystem;
```

Comment:
I'm not a fan, but I can clean it up later.

Reply:
I've removed it.